<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>snowgui::EventDisplayWidget</class>
 <widget class="QWidget" name="snowgui::EventDisplayWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1" colspan="5">
    <widget class="QLineEdit" name="filterField"/>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="toLabel">
     <property name="text">
      <string>To:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="fromLabel">
     <property name="text">
      <string>From:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QPushButton" name="timeSelectButton">
     <property name="text">
      <string>Select</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="filterLabel">
     <property name="text">
      <string>Filter:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="7">
    <widget class="QTableWidget" name="eventTable">
     <property name="columnCount">
      <number>8</number>
     </property>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
    </widget>
   </item>
   <item row="2" column="6">
    <widget class="QPushButton" name="filterButton">
     <property name="text">
      <string>Filter</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QDateTimeEdit" name="fromTime">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="displayFormat">
      <string>dd.MM.yy HH:mm:ss</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="4" colspan="2">
    <widget class="QDateTimeEdit" name="toTime">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="displayFormat">
      <string>dd.MM.yy HH:mm:ss</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
